### DISCUSSION TITLE
github-actions[bot] and comments

### DISCUSSION THREAD

**Discussion by XhmikosR** [community/community#25589] [Actions]
<p>This is something that might not be related to GitHub, but to <a href="https://github.com/coverallsapp/github-action" rel="nofollow noopener">coverallsapp/github-action.</a></p>
<p>I’m trying to find more info how one can control if the github-actions[bot] should leave a comment. The reason is that currently, even if one has disabled PR comments through Coveralls dashboard, there’s always a comment from github-actions[bot]. See the upstream issue <a href="https://github.com/coverallsapp/github-action/issues/5" rel="nofollow noopener">here</a>.</p>
<p>Being that I cannot get anyone from Coveralls to look into it, I thought I’d ask here.</p>
<p>Thanks in advance!</p>

**Comment by ethomson** [comment]
<p>I’m sorry that the Coveralls team has not been responsive, but ultimately their action is creating an issue comment.  You could, perhaps, create a personal access token that doesn’t have permissions to comment, but the easiest solution would be for them to add an option to the action about whether it comments or not.</p>

**Comment by XhmikosR** [comment, marked as answer]
<p>It seems this was fixed!</p>
<p><a href="https://github.com/coverallsapp/github-action/issues/5#issuecomment-553555754" rel="nofollow noopener">https://github.com/coverallsapp/github-action/issues/5#issuecomment-553555754</a></p>
